npm/cypress-schematic/src/schematics/ng-generate/cypress-ct-tests/schema.json

```diff
@@ -0,0 +1,25 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema",
+  "$id": "cypress-schematics-e2e-spec",
+  "title": "Cypress E2E Spec Options Schema",
+  "type": "object",
+  "properties": {
+    "project": {
+      "type": "string",
+      "description": "The name of the project.",
+      "alias": "p",
+      "$default": {
+        "$source": "projectName"
+      }
+    },
+    "createSpecs": {
```

Comment:
What is the purpose of this arg? If they say no, the schematic doesn't do anything

Reply:
Good catch. I'm removing this.